website/static/js/projectorganizer.js

```diff
@@ -1280,6 +1291,40 @@ ProjectOrganizer.prototype = {
     _initGrid: function () {
         this.grid = new Treebeard(this.options);
         return this.grid;
+    },
+    legend: function(node) {        
+        var self = this;
+        var showLegend = function() {
+            var keys = Object.keys(projectOrganizerCategories);
+            var legend = new Legend(keys.map(function(key) {
```

Comment:
The `Legend` constructor is a little unintuitive. the `new` isn't really necessary, since you don't need to really construct a legend object; you need to render the view for the legend.

Possible refactor:

```js
var legend = require('js/components/legend');

var legendView = legend.view(data, options);
self.grid.tbController.modal.update(legendView);
```